docs/en/deploy/Running-Alluxio-On-Yarn.md

```diff
@@ -6,6 +6,14 @@ group: Deploying Alluxio
 priority: 6
 ---
 
+TODO: Expand the explanation of why we prefer to run alongside YARN instead of on top. Relevant
+mailing list thread: https://groups.google.com/forum/#!topic/alluxio-users/KEvBE9AiJlE
```

Comment:
do we need to talk about setting up classpath for YARN applications to include Alluxio client jar some where ?

Reply:
+1, if users are reading this doc, they are probably also planning to run their applications on YARN as well. We should make sure to also bring this up in the compute applications sections, since it's needed even if Alluxio isn't deployed on YARN or co-located with the YARN cluster.